Compass/src/Controller/AwardsController.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  AwardsController.swift
+//  Compass
+//
+//  Created by Ismael Alonso on 7/14/16.
+//  Copyright © 2016 Tennessee Data Commons. All rights reserved.
+//
+
+import UIKit
+import Just
+import ObjectMapper
+
+
+class AwardsController: UITableViewController{
+    private var awards: [Award] = [Award]();
+    
+    override func viewDidLoad(){
+        Just.get(API.getAwardsUrl(), headers: SharedData.user.getHeaderMap()){ (response) in
+            if (response.ok){
+                let result = String(data: response.content!, encoding:NSUTF8StringEncoding)!;
+                let aa = Mapper<ParserModels.AwardArray>().map(result)!;
+                if (aa.awards != nil){
+                    self.awards = aa.awards!;
+                }
+                var paths = [NSIndexPath]();
+                for i in 0...self.awards.count-1{
```

Comment:
We should do `for i in 0..<self.awards.count` instead of ` 0...self.awards.count-1`, because the latter will result in the following error:

> fatal error: Can't form Range with end < start